pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -268,6 +269,14 @@ async def run(self):
             except CalledProcessError as ex:
                 _LOGGER.warning(f"Unable to move {impetus} advisory {advisory} to QE: {ex}")
 
+        _LOGGER.info("Check failed greenwave tests ...")
+        self._slack_client.bind_channel(self.release_name)
+        for advisory in [advisories["image"], advisories["extras"], advisories["metadata"]]:
+            test_result = get_advisory_greenwave(advisory)
+            if test_result != []:
+                _LOGGER.warning(f"Some greenwave tests on {advisory} failed with {test_result}")
+                await self._slack_client.say(f"Some greenwave tests failed on {advisory} @release-artists")
```

Comment:
I'd really like to have a link here, so I can be lazy.
```suggestion
                await self._slack_client.say(f"Some greenwave tests failed on https://errata.devel.redhat.com/advisory/{advisory}/test_run/greenwave_cvp @release-artists")
```